src/applications/mhv/secure-messaging/tests/components/MessageListComponents/folderHeader.unit.spec.jsx

```diff
@@ -30,41 +38,275 @@ describe('FolderHeader component in custom folder', () => {
       },
     },
   };
-  const setup = (state = initialState) => {
+  const initialPath = `/folders/${customFolder.folderId}`;
+  const setup = (
+    state = initialState,
+    path = initialPath,
+    threadCount = threadList,

```

Comment:
if the `propType` above will be changed, you would probably need a tweak here too

Reply:
This is updated.